code/PostgreSQLDatabase.php

```diff
@@ -214,7 +214,7 @@ public function connect($parameters)
         }
         $this->schemaOriginal = $parameters['schema'];
 
-        // Ensure that driver is available (required by PDO)
+        // Ensure that driver is available
         if (empty($parameters['driver'])) {
             $parameters['driver'] = $this->getDatabaseServer();
         }
```

Comment:
If that was only required by PDO, we should remove it now.

Reply:
I can't 100% verify that this comment is actually accurate and won't cause an edge case failure in non-pdo postresql.  Since we know it works with this here now, I think we should just keep it as is

Reply:
Fair enough